Rectangle/MultiWindow/MultiWindowManager.swift

```diff
@@ -113,6 +113,21 @@ class MultiWindowManager {
         }
     }
 
+    private struct CascadeActiveAppWindows {
+        var right = false
+        var bottom = false
+        var n = 0
+        var size = CGSizeZero
+
+        init(windowFrame: CGRect, screenFrame: CGRect, n: Int, size: CGSize, delta: CGFloat) {
+            right = windowFrame.midX > screenFrame.midX
+            bottom = windowFrame.midY > screenFrame.midY
+            self.n = n
+            let m = CGSize(width: screenFrame.width - CGFloat(n - 1) * delta, height: screenFrame.height - CGFloat(n - 1) * delta)
+            self.size = CGSize(width: min(size.width, m.width), height: min(size.height, m.height))
```

Comment:
n and m here take some reasoning to figure out what they mean. Can you think of better names for them?